crates/router/src/core/payouts.rs

```diff
@@ -684,7 +696,34 @@ pub async fn payouts_list_core(
             )
             .await
         {
-            Ok(payout_attempt) => Some(Ok((payouts, payout_attempt))),
+            Ok(payout_attempt) => {
+                match db
+                    .find_customer_by_customer_id_merchant_id(
+                        &payouts.customer_id,
+                        merchant_id,
+                        &key_store,
+                    )
+                    .await
+                {
+                    Ok(customer) => Some(Ok((payouts, payout_attempt, customer))),
+                    Err(error) => {
+                        if matches!(
+                            error.current_context(),
+                            storage_impl::errors::StorageError::ValueNotFound(_)
+                        ) {
+                            logger::warn!(
+                                ?error,
+                                "customer missing for customer_id : {}",
```

Comment:
Will it be wise to log errors with info suggesting its payout flow.

Reply:
Logs are already scoped with request IDs and flow type, right? We don't need to add more context about payouts in this log line if these are already fetched using request IDs